eml/components/explore/ExploreCard.jsx

```diff
@@ -1,34 +1,224 @@
-import { View, Text, Image, Pressable } from 'react-native'
-import React from 'react'
-import { useFonts, VarelaRound_400Regular } from '@expo-google-fonts/dev'
-import { useNavigation } from '@react-navigation/native'
-import { AppLoading } from 'expo-app-loading'
+import React, { useEffect } from "react";
+import { View, Text, Image, Pressable } from "react-native";
+import Collapsible from "react-native-collapsible";
+import { useNavigation } from "@react-navigation/native";
+import { useFonts, VarelaRound_400Regular } from "@expo-google-fonts/dev";
```

Comment:
Not needed (when the leftover function has been removed)

Reply:
video stream group again ...

Reply:
@Educado-App/video-streaming-team